app/uk/gov/hmrc/brm/utils/BrmException.scala

```diff
@@ -48,10 +48,16 @@ trait BRMException extends Controller {
 
   def forbiddenPF(message: String)(implicit payload: Payload) : PartialFunction[Throwable, Result] = {
     case Upstream4xxResponse(m, FORBIDDEN, _, _) if m.contains("INVALID_DISTRICT_NUMBER") =>
-      logException(Some(message), Some(s"Forbidden returned from NRS message"), BAD_REQUEST)
+      logException(Some(message), Some(m), BAD_REQUEST)
       BadRequest
```

Comment:
@adamconder Ya we can send 200 false. Also if we get information what happens for bad request in dfs side.